src/fsharp/NicePrint.fs

```diff
@@ -1736,7 +1764,7 @@ module private TastDefinitionPrinting =
 
 //--------------------------------------------------------------------------
 
-module private InferredSigPrinting = 
```

Comment:
What if these are made internal instead of private?  Does Fable support internal modules? thanks